### DISCUSSION TITLE
Truncate component

### DISCUSSION THREAD

**Discussion by mindytom** [twilio-labs/paste#1695] [General] [answered]
I was looking over the truncate component guideline and read: 
> Truncation should NOT be used on page headers, titles, labels, error messages, validation messages, notifications, or SIDs.

I spoke with @katieporter to get clarification on what to do when Friendly Names are super long and they happen to be used as the page header, titles, etc. Moving forward I'll be using truncation for these use cases. 
<img width="1792" alt="Screen Shot 2021-07-26 at 1 15 14 PM" src="https://user-images.githubusercontent.com/65037539/127060604-c5b82881-3511-41df-88c8-044dc40f3779.png">

<img width="1990" alt="Screen Shot 2021-07-26 at 2 08 32 PM" src="https://user-images.githubusercontent.com/65037539/127060775-341be8ba-b46c-4614-9aaa-feb7a29ed62b.png">

**Comment by TheSisb** [comment, marked as answer]
Hi @mindytom,

Thanks for pointing that out, the Truncate component doc page seems a little strict in that regard. I think your use case is fine, and I will go ahead and update the doc page to say something like:

> Good use cases for truncation include:
> - Breadcrumbs
> - Long URL links
> - Long friendly names
> 
> Truncation should generally not be used for page titles, labels, error messages, validation messages, notifications, or SIDs. A notable exception is when a long friendly name is used as a page title.

**Reply by TheSisb** [reply]
PR is up: https://github.com/twilio-labs/paste/pull/1696

**Reply by TheSisb** [reply]
Merged!
